### DISCUSSION TITLE
Are there any alternatives to the deprecated classes and methods in jedis 3.10.0?

### DISCUSSION THREAD

**Discussion by libailb** [xetorthio/jedis#3878] [Q&A]
We upgraded from jedis version 3.6.3 to 3.10.0 and found that many classes and methods have been deprecated, including but not limited to JedisShardInfo, ShardedJedis, Client, ShardedJedisPipeline, ShardedJedisPipeline, and ShardInfo. No relevant replacements were found on the jedis website. illustrate. Can you explain the replacement implementation?

**Comment by sazzad16** [comment]
Please check https://github.com/redis/jedis/discussions/3469 and https://github.com/redis/jedis/discussions/3513.
